agent/tool-scripts/prometheus-metrics

```diff
@@ -115,6 +114,11 @@ tool_stderr_file=$tool_output_dir/$tool-stderr.txt
 tool_log=$tool_output_dir/tool.log
 ## create directory to save the keys from openshift master, hosts
 mkdir -p /run/pbench/keys
+## check if the inventory variable is set
+if [[ -z "$inventory" ]] || [[ ! -s "$inventory" ]]; then
+	echo "Please set the inventory=/path/to/inventory while registering the tool and make sure file exists and is not empty"
```

Comment:
It would be helpful I think if you mentioned *what* file is missing or empty in the error message.